examples/js/web-elements-app/src/app/app.element.ts

```diff
@@ -452,7 +453,7 @@ const user = {
     isAnonymous: false
 }
 
-const options = { enableEdgeDB: true }
+const options = { enableEdgeDB: true, logLevel: 'error' as DVCDefaultLogLevel }
```

Comment:
casting the logLevel value like this can't be a requirement for Typescript users. Using the string directly should be possible